x-pack/plugin/transform/src/main/java/org/elasticsearch/xpack/transform/action/TransportPreviewTransformAction.java

```diff
@@ -112,6 +115,7 @@ public TransportPreviewTransformAction(
 
     @Override
     protected void doExecute(Task task, Request request, ActionListener<Response> listener) {
+        TaskId parentTaskId = new TaskId(clusterService.getNodeName(), task.getId());
```

Comment:
This should use the node ID, not the node name.

(I guess you copied this from one of the ML PRs. If you look at the current code you’ll see there was a followup to fix this mistake.)

Reply:
Ooops, that's true.

Done.